src/SFA.DAS.AAN.Hub.Jobs/Configuration/ApplicationConfiguration.cs

```diff
@@ -6,6 +6,7 @@ public class ApplicationConfiguration
 {
     public Uri ApprenticeAanBaseUrl { get; set; }
     public Uri EmployerAanBaseUrl { get; set; }
+    public Uri ApprenticeAccountsApiBaseUrl { get; set; }
```

Comment:
I was expecting a property of type `InnerApiConfiguration` here with name `ApprenticeAccountsApiConfiguration` which matches the config name.